### PR TITLE
Add Security Agent flare creation steps

### PR DESCRIPTION
**Pull request by tiphainePLQ**
Slack discussion with Sylvain https://dd.slack.com/archives/C053TE945S8/p1713794209453469?thread_ts=1713520873.514109&cid=C053TE945S8

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by safchain — APPROVED** [review]

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 1]
```diff
@@ -256,6 +256,17 @@ In the task definition, replace the "workload" container with the following:
             ],
 {{< /code-block >}}
 
+## Next steps
+If you are still running into an issue, please [open a ticket][10] to Support with a `security-agent` flare attached to the latter.
```
Comment (rtrieu) [inline content/en/security/cloud_security_management/setup/fargate.md:260]: Hi @tiphainePLQ , is this more of what you meant?

```suggestion
If you still have issues with the configuration, please [contact Datadog support][10] for help and attach the `security-agent` flare to the ticket.
```

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 2]
```diff
@@ -256,6 +256,17 @@ In the task definition, replace the "workload" container with the following:
...
+Find below the step to create a `security-agent` flare according to your deployment:
```
Comment (rtrieu) [inline content/en/security/cloud_security_management/setup/fargate.md:261]:
```suggestion
Use the appropriate command below to create a `security-agent` flare based on your deployment:
```

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 3]
```diff
@@ -266,3 +277,5 @@ In the task definition, replace the "workload" container with the following:
 [7]: /resources/json/datadog-agent-cws-ecs-fargate.json
 [8]: /integrations/faq/integration-setup-ecs-fargate/?tab=rediswebui
 [9]: https://app.datadoghq.com/logs
+[10]: https://docs.datadoghq.com/help/
```
Comment (rtrieu) [inline content/en/security/cloud_security_management/setup/fargate.md:280]:
```suggestion
[10]: /help/
```

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 4]
```diff
@@ -266,3 +277,5 @@ In the task definition, replace the "workload" container with the following:
...
+[11]: https://docs.datadoghq.com/agent/troubleshooting/send_a_flare/?tab=agentv6v7#ecs-fargate
```
Comment (rtrieu) [inline content/en/security/cloud_security_management/setup/fargate.md:281]:
```suggestion
[11]: /agent/troubleshooting/send_a_flare/?tab=agentv6v7#ecs-fargate
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some suggestions for your review before publishing. Thanks!

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**